src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -485,6 +547,10 @@ class DocAnnotator extends Annotator {
             this.highlightThrottleHandle = null;
         }
 
+        if (!this.permissions.canAnnotate) {
```

Comment:
i don't think this is needed?

Reply:
I'd say it's unnecessary to remove it, as we don't want to be removing event listeners that don't exist. I'm not sure what our standard is for that. @jeremypress @tonyjin @MinhHNguyen Thoughts?

Reply:
It's safe to remove event listeners that don't exist. I'd personally prefer fewer lines of code over adding a guard before removing event listeners but I don't think this is something we need to standardize on.